app/models/resource_evaluation.rb

```diff
@@ -14,4 +14,6 @@ class ResourceEvaluation < ApplicationRecord
   belongs_to :user
 
   validates :user, uniqueness: { scope: [:resource] }
+  validates :evaluation, numericality: { only_integer: true, in: (0..5),
```

Comment:
I think this validation should be reflected in the schema of the database, therefore a migration file is needed. I found this [example](https://boringrails.com/tips/rails-check-constraints-database-validations#:~:text=As%20of%20Rails%206.1%2C%20you%20can%20specific%20check%20constraints%20straight%20from%20a%20migration.)

Reply:
Thanks! I will make the migration first.